vmdk_plugin/utils/fs/fs.go

```diff
@@ -37,6 +37,9 @@ const pciAddrLen = 10                     // Length of PCI dev addr
 // FstypeDefault contains the default FS when not specified by the user
 const FstypeDefault = "ext4"
 
+// BinSearchPaths contains search paths for host binaries
+var BinSearchPaths = [4]string{"/bin", "/sbin", "/usr/bin", "/usr/sbin"}
```

Comment:
>This may not be needed here, why not have a "default" search path in the default config used by the plugin and that can be used as the search path for all external executables that the plugin runs. The plugin behavior can be changed any time.

I do think that is probably better to have it in the config file.

Please correct me if i am wrong, but we hold no reference from the config instance created in main.go (where the configfile can be other than the default), and from fs.go we would only be able to load it with config.DefaultConfigPath, this file can potentially be different from the user specified - If this is the case, IMO the required changes are beyond the scope of this PR.

>Even if we want to have a list here maybe a slice can be used vs. an array - []string{"/bin", "/sbin", "/usr/bin", "/usr/sbin"}.

It's not really needed, but its not a problem :)